### PR TITLE
2/2 Update templates to new NI Branding

### PR DESCRIPTION
**Pull request by doczhivago**
[x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

[x] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.

[x] I've added tests applicable for this pull request

What does this Pull Request accomplish?
Update templates to correctly reference updating branding for NI and National Instrument Corporation where applicable.

List issues fixed by this Pull Request below, if any.
First PR (2/2) to resolves Issue https://github.com/ni/nimi-python/issues/1490

What testing has been done?
Tests pass on local machine, no additional tests needed to be written.

### CODE REVIEW

File: docs/nidigital/class.rst [hunk 1]
```diff
@@ -4007,7 +4007,7 @@ instrument_manufacturer
 
     .. py:attribute:: instrument_manufacturer
 
-        Returns a string ("National Instruments") that contains the name of the manufacturer of the digital pattern instrument.
+        Returns a string ("NI") that contains the name of the manufacturer of the digital pattern instrument.
```
Comment (sbethur) [inline docs/nidigital/class.rst:4010]: We didn't update the driver to return `NI` as part of brand identity updates; it still returns `National Instruments`.

File: docs/nidigital/class.rst [hunk 2]
```diff
@@ -5219,7 +5219,7 @@ specific_driver_vendor
 
     .. py:attribute:: specific_driver_vendor
 
-        Returns a string ("National Instruments") that contains the name of the vendor that supplies the NI-Digital Pattern Driver.
+        Returns a string ("NI") that contains the name of the vendor that supplies the NI-Digital Pattern Driver.
```
Comment (sbethur) [inline docs/nidigital/class.rst:5222]: We didn't update the driver to return `NI` as part of brand identity updates; it still returns `National Instruments`.

**Review by sbethur — CHANGES_REQUESTED** [review]

**Comment by sbethur** [comment]
PR build failed due to ensure_codegen_up_to_date check:
```
The following code generated files do not match what is commited to Git. Run codegen and include the generated files in the PR.
 M docs/nidigital/class.rst
 M generated/nifake/README.rst
```

Please run `tox -e clean` followed by `tox` locally and include the updated generated files in the PR.

(Thanks @ayesham-729 for adding the ensure_codegen_up_to_date check!!)

File: docs/nidigital/class.rst (hunk 2 above)
Comment (doczhivago) [reply docs/nidigital/class.rst]: Updated.

File: src/nidmm/examples/nidmm_fetch_waveform.py [hunk 3]
```diff
@@ -21,7 +21,7 @@ def example(resource_name, options, function, range, points, rate):
 
 def _main(argsv):
     parser = argparse.ArgumentParser(description='Performs a waveform acquisition using the NI-DMM API.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of a National Instruments Digital Multimeter.')
+    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of an NI Digital Multimeter module.')
```
Comment (marcoskirsch) [inline src/nidmm/examples/nidmm_fetch_waveform.py:24]: remove "module". If anything we should say "instrument" per documentation guidelines but in this case it's not needed.

File: src/nidmm/examples/nidmm_measurement.py [hunk 4]
```diff
@@ -14,7 +14,7 @@ def example(resource_name, option_string, function, range, digits):
 def _main(argsv):
     supported_functions = list(nidmm.Function.__members__.keys())
     parser = argparse.ArgumentParser(description='Performs a single measurement using the NI-DMM API.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of a National Instruments Digital Multimeter.')
+    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of an NI Digital Multimeter module.')
```
Comment (marcoskirsch) [inline src/nidmm/examples/nidmm_measurement.py:17]: remove "module"

File: src/nidmm/examples/nidmm_multi_point_measurement.py [hunk 5]
```diff
@@ -15,7 +15,7 @@ def example(resource_name, options, function, range, digits, samples, triggers):
 
 def _main(argsv):
     parser = argparse.ArgumentParser(description='Performs a multipoint measurement using the NI-DMM API.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of a National Instruments Digital Multimeter.')
+    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of an NI Digital Multimeter module.')
```
Comment (marcoskirsch) [inline src/nidmm/examples/nidmm_multi_point_measurement.py:18]: remove "module"

File: src/nifgen/examples/nifgen_arb_waveform.py [hunk 6]
```diff
@@ -27,7 +27,7 @@ def example(resource_name, options, samples, gain, offset, gen_time):
 
 def _main(argsv):
     parser = argparse.ArgumentParser(description='Continuously generates an arbitrary waveform.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of a National Instruments Arbitrary Waveform Generator')
+    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of an NI Arbitrary Waveform Generator module.')
```
Comment (marcoskirsch) [inline src/nifgen/examples/nifgen_arb_waveform.py:30]: remove "module"

File: src/nifgen/examples/nifgen_script.py [hunk 7]
```diff
@@ -153,7 +153,7 @@ def example(resource_name, options, shape, channel):
 
 def _main(argsv):
     parser = argparse.ArgumentParser(description='Generate different shape waveforms.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of a National Instruments Arbitrary Waveform Generator')
+    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of an NI Arbitrary Waveform Generator module.')
```
Comment (marcoskirsch) [inline src/nifgen/examples/nifgen_script.py:156]: remove "module"

File: src/nifgen/examples/nifgen_standard_function.py [hunk 8]
```diff
@@ -17,7 +17,7 @@ def example(resource_name, options, waveform, frequency, amplitude, offset, phas
 def _main(argsv):
     supported_waveforms = list(nifgen.Waveform.__members__.keys())[:-1]  # no support for user-defined waveforms in example
     parser = argparse.ArgumentParser(description='Generates the standard function.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of a National Instruments Function Generator')
+    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of an NI Function Generator module.')
```
Comment (marcoskirsch) [inline src/nifgen/examples/nifgen_standard_function.py:20]: remove "module"

File: src/niscope/examples/niscope_fetch.py [hunk 9]
```diff
@@ -21,7 +21,7 @@ def example(resource_name, channels, options, length, voltage):
 
 def _main(argsv):
     parser = argparse.ArgumentParser(description='Acquires one record from the given channels.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of a National Instruments Digitizer')
+    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of an NI Digitizer module.')
```
Comment (marcoskirsch) [inline src/niscope/examples/niscope_fetch.py:24]: remove "module"

File: src/niscope/examples/niscope_fetch_forever.py [hunk 10]
```diff
@@ -44,7 +44,7 @@ def example(resource_name, options, total_acquisition_time_in_seconds, voltage,
 
 def _main(argsv):
     parser = argparse.ArgumentParser(description='Fetch more samples than will fit in memory.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of a National Instruments Digitizer')
+    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of an NI Digitizer module.')
```
Comment (marcoskirsch) [inline src/niscope/examples/niscope_fetch_forever.py:47]: remove "module"

File: src/niscope/examples/niscope_read.py [hunk 11]
```diff
@@ -20,7 +20,7 @@ def example(resource_name, channels, options, length, voltage):
 
 def _main(argsv):
     parser = argparse.ArgumentParser(description='Acquires one record from the given channels.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of a National Instruments Digitizer')
+    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of an NI Digitizer module.')
```
Comment (marcoskirsch) [inline src/niscope/examples/niscope_read.py:23]: remove "module"

File: src/niswitch/examples/niswitch_connect_channels.py [hunk 12]
```diff
@@ -18,7 +18,7 @@ def example(resource_name, channel1, channel2, topology, simulate):
 
 def _main(argsv):
     parser = argparse.ArgumentParser(description='Performs a connection with NI-SWITCH Channels.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of a National Instruments Switch.')
+    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of an NI-SWITCH module.')
```
Comment (marcoskirsch) [inline src/niswitch/examples/niswitch_connect_channels.py:21]: remove "module"

File: src/niswitch/examples/niswitch_get_device_info.py [hunk 13]
```diff
@@ -36,8 +36,8 @@ def example(resource_name, topology, simulate, device, channel, relay):
 
 
 def _main(argsv):
-    parser = argparse.ArgumentParser(description='Prints information for the specified National Instruments Switch module.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of a National Instruments Switch.')
+    parser = argparse.ArgumentParser(description='Prints information for the specified NI-SWITCH module.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
```
Comment (marcoskirsch) [inline src/niswitch/examples/niswitch_get_device_info.py:39]: remove "module"

File: src/niswitch/examples/niswitch_get_device_info.py [hunk 14]
```diff
@@ -36,8 +36,8 @@ def example(resource_name, topology, simulate, device, channel, relay):
...
+    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of an NI-SWITCH module.')
```
Comment (marcoskirsch) [inline src/niswitch/examples/niswitch_get_device_info.py:40]: remove "module"

File: src/niswitch/examples/niswitch_relay_control.py [hunk 15]
```diff
@@ -16,7 +16,7 @@ def example(resource_name, topology, simulate, relay, action):
 
 def _main(argsv):
     parser = argparse.ArgumentParser(description='Performs relay control with NI-SWITCH relays.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of a National Instruments Switch.')
+    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of an NI-SWITCH module.')
```
Comment (marcoskirsch) [inline src/niswitch/examples/niswitch_relay_control.py:19]: remove "module"

**Review by marcoskirsch — CHANGES_REQUESTED** [review]

File: src/nidmm/examples/nidmm_fetch_waveform.py (hunk 3 above)
Comment (doczhivago) [reply src/nidmm/examples/nidmm_fetch_waveform.py]: Update

File: src/niswitch/examples/niswitch_connect_channels.py [hunk 16]
```diff
@@ -18,7 +18,7 @@ def example(resource_name, channel1, channel2, topology, simulate):
 
 def _main(argsv):
     parser = argparse.ArgumentParser(description='Performs a connection with NI-SWITCH Channels.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of a National Instruments Switch.')
+    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of an NI-SWITCH.')
```
Comment (marcoskirsch) [inline src/niswitch/examples/niswitch_connect_channels.py:21]: "NI-SWITCH" is the name of a software product.
An "NI switch" is a switch made by NI.

Looking at all these, I realize we should not capitalize the instrument type unless it's an initialism (DMM, SMU).

Can you fix please?

**Review by marcoskirsch — CHANGES_REQUESTED** [review]

File: src/niswitch/examples/niswitch_connect_channels.py (hunk 16 above)
Comment (doczhivago) [reply src/niswitch/examples/niswitch_connect_channels.py]: Updated

File: src/nidmm/examples/nidmm_fetch_waveform.py [hunk 17]
```diff
@@ -21,7 +21,7 @@ def example(resource_name, options, function, range, points, rate):
 
 def _main(argsv):
     parser = argparse.ArgumentParser(description='Performs a waveform acquisition using the NI-DMM API.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of a National Instruments Digital Multimeter.')
+    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of an NI Digital Multimeter.')
```
Comment (marcoskirsch) [inline src/nidmm/examples/nidmm_fetch_waveform.py:24]: digitizer, digital multimeter, arbitrary waveform generator, function generator should follow the same rules and not be capitalized

**Review by marcoskirsch — CHANGES_REQUESTED** [review]

File: src/nidmm/examples/nidmm_fetch_waveform.py (hunk 17 above)
Comment (doczhivago) [reply src/nidmm/examples/nidmm_fetch_waveform.py]: Updated.

**Review by marcoskirsch — APPROVED** [review]

**Comment by sbethur** [comment]
One more reference to "National Instruments" that needs to be updated - In the docstring of `Session` class. e.g. https://github.com/ni/nimi-python/blob/d170112dfb80aaa18367bf95a584fe19d5aaae3d/generated/nidcpower/nidcpower/session.py#L5709

It comes from https://github.com/ni/nimi-python/blob/d170112dfb80aaa18367bf95a584fe19d5aaae3d/src/nidcpower/metadata/config.py#L50

(Applies to other modules as well)

**Comment by sbethur** [comment]
One more instance - https://github.com/ni/nimi-python/blob/fdb9fe106bc18acc6cfac18d56b1b489743f243f/src/nimodinst/metadata/functions.py#L148

**Comment by sbethur** [comment]
More: 
- https://github.com/ni/nimi-python/blob/cdd26263b6ce61771f75d1e52077d06d564cf085/src/niscope/metadata/functions.py#L2470
- Multiple instances in https://github.com/ni/nimi-python/blob/master/src/nidmm/metadata/functions.py

**Comment by sbethur** [comment]
Please note any changes to metadata should be replicated in the internal metadata source.

**Review by sbethur — APPROVED** [review]

**Pull request merged**